serverConfig-example.js

```diff
@@ -8,8 +8,16 @@ exports.callbackURL = 'your callbackURL [http://123.123.123.123:3000/auth/github
 exports.gitHooksCallbackURL = 'your URL for Git to send hooks notifications [http://123.123.123.123:8080]';
 // when running on a LAN with a router, the URLs above need your public IP (can get from google), GitHub will not register a hook to 'localhost'
 // the localHost value below will either be your public IP address, or potentially your unique IP within your personal LAN
-exports.localHost = 'your local host (could be localhost or an ip address) [123.123.123.123 OR ~192.168.X.X]';
+exports.localHost = 'localhost';
```

Comment:
Did you mean to add all this to the 'serverConfig-example' file?  Shouldn't we keep this generic as a resource for others?